src/data/resolvers/colony.ts

```diff
@@ -123,10 +127,12 @@ export const getProcessedColony = async (
         colonyDisplayName = null,
         colonyAvatarHash = null,
         colonyTokens = [],
+        colonySafes = [],
```

Comment:
For V2 metadata in IPFS, will safe data be added to a ColonyMetadata Object (as listed here) or should we create a new MetadataType?

Reply:
As far as I know and from what I've seen in recent implementations of the safe dialogs, it has been very convenient for us to have it with the colony's metadata